modules/simple-payments/simple-payments.php

```diff
@@ -155,7 +154,37 @@ function parse_shortcode( $attrs, $content = false ) {
 		return $this->output_shortcode( $data );
 	}
 
-	function ignore_shortcode() { return; }
+	function output_admin_warning( $data ) {
+		if ( ! current_user_can( 'manage_options' ) ) {
+			return;
+		}
+		$css_prefix = self::$css_classname_prefix;
+
+		$support_url = ( defined( 'IS_WPCOM' ) && IS_WPCOM )
+			? 'https://support.wordpress.com/simple-payments/'
+			: 'https://jetpack.com/support/simple-payment-button/';
+
+		$warning = sprintf(
+			wp_kses(
```

Comment:
Why do we need `wp_kses` here? The only variable that we pass to the string is `$support_url`, and it's already sanitized with `esc_url`. All of the other tags are hard-coded and completely controlled by us.

Reply:
I'm not sure the localized strings are completely controlled by us.
What if a third party translator (can be an OSS contributor, an external localization company, a disgruntled a11n 😄, etc.) sneaked in some unexpected attributes to our link, and we didn't check the .po file because 🤷‍♂️ ?

Reply:
I completely missed the translation part while reading. You are right, we need to keep `wp_kses` here. :)

Reply:
Another option would be to treat this as two separate strings for translation, since I think they could be translated independently. That way we could use `esc_html__` and move the `a` tag part outside of the translation parameter.